liota/core/package_manager.py

```diff
@@ -373,10 +379,18 @@ def run(self):
 
             # Switch on message content (command), determine what to do
             command = msg[0]
+            if command in ["unload", "delete", "list", "stat", "unload_all",\
+                           "unload_all", "terminate"]:
+                # currently, write 'Success' to the named pipe once
+                #     successfully receives these commands
+                with open(self._rsp_pipe_file, "w+") as fp:
```

Comment:
If the open fails we need to do something... log an error?

Reply:
added